src/utils/RemoveNullProps.ts

```diff
@@ -0,0 +1,15 @@
+/******************
+ * Null action properties can be assigned to the ShareClasses when cancelling edits
+ * This is fail safe to ensure null actions are not included in the comparison
```

Comment:
Good comment but I think it belongs in the store code where you use it.

Here, this is a generic util, which doesn't know about share classes. Use a generic description instead.